include/image-commands.mk

```diff
@@ -329,6 +330,8 @@ define Build/sysupgrade-tar
 		--board $(if $(BOARD_NAME),$(BOARD_NAME),$(DEVICE_NAME)) \
 		--kernel $(call param_get_default,kernel,$(1),$(IMAGE_KERNEL)) \
 		--rootfs $(call param_get_default,rootfs,$(1),$(IMAGE_ROOTFS)) \
+		--dtb $(call param_get_default,dtb,$(1),$(KDIR)/image-$(firstword $(DEVICE_DTS)).dtb) \
+		--ramdisk $(call param_get_default,ramdisk,$(1),$(IMAGE_KERNEL).fakehdr) \
```

Comment:
This introduces changes to all devices in OpenWrt using that line. Please don't do that (unless you think that _all_ devices require it and you have tested a fair share of them. In that case, it should be a separate commit BTW.)
If you need this for a specific device, please create a specific recipe for it (in the relevant target's image/Makefile).
Same goes for the addition above in append-uImage-fakehdr

Reply:
OK, let me try to rework the patch to make it device specific.  If more than one device ends up needing it, we can generalize it.

Reply:
Changes to `include/image-commands.mk` have been fully dropped.